src/runtime/cmd/kata-runtime/kata-check_amd64_test.go

```diff
@@ -18,6 +19,8 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const blacklistModuleConf = "/etc/modprobe.d/blacklist-kata-kernel-modules.conf"
```

Comment:
s/blacklist/denylist/g

And also everywhere in this patch.

Reply:
Fixed.